webapp/channels/src/components/emoji_picker/components/emoji_picker_skin.tsx

```diff
@@ -144,6 +145,8 @@ export class EmojiPickerSkin extends React.PureComponent<Props, State> {
                         className='skin-tones__close-icon style--none'
                         onClick={() => this.hideSkinTonePicker(this.props.userSkinTone)}
                         aria-label={closeButtonLabel}
+                        aria-expanded={this.state.pickerExtended}
+                        tabIndex={0}
```

Comment:
Shouldn't this button have a `aria-controls` attribute as well?
Also, add the `aria-label` and `role` to the choices wrapper.